File: utils/judge.go
package utils

import "fmt"

func eqArray(source []int, target []int) bool {
	if len(source) != len(target) {
		return false
	}
	for i, v := range source {
		if target[i] != v {
			return false
		}
	}
	return true
}

func JudgeWithArray(problem string, source []int, target []int) {
	if eqArray(source, target) {
		fmt.Printf("[%s] Accept! \n", problem)
		return
	}
	fmt.Printf("[%s] Wrong!, Answer: %v / Expected: %q \n", problem, source, target)
}

func JudgeWithValue() {}

func JudgeWithLinkList() {}

File: utils/judge.js
function eqArr(source, target) {
    if (source.length !== target.length) return false
    for(let i = 0; i < source.length; i ++) {
        if (source[i] !== target[i]) {
            return false
        }
    }
    return true
}


module.exports = {
    judgeWithArray(problem, source, target) {
        if (eqArr(source, target)) {
            console.log("[%s] Accept!", problem) 
            return
        }
        console.error("[%s] Wrong!, Answer: %s / Expected: %s", problem, source, target)
    },
    judgeWithLinkList(problem, source, target) {
        
    },
    judgeWithValue(problem, source, target) {

    }

}